experimental/cc_slurm_nhc/cc_slurm_nhc/specs/default/cluster-init/files/configure_nhc.sh

```diff
@@ -116,7 +115,7 @@ function update_slurm_prolog_epilog() {
          echo "Epilog=/sched/scripts/epilog.sh" >> $SLURM_CONF
       fi
    fi
-   echo "/sched/scripts/$script $prolog_epilog $PROLOG_RUN_NHC" >> /sched/scripts/${prolog_epilog}.sh
+   echo "/sched/scripts/$script $prolog_epilog" >> /sched/scripts/${prolog_epilog}.sh
```

Comment:
wait_for_nhc.sh is expecting 2 arguments.

Reply:
That is correct. I have reverted the change